src/components/AllRequests.jsx

```diff
@@ -10,7 +10,7 @@ const AllRequests = () => {
 
   useEffect(() => {
     getRequests(dispatch);
-  }, []);
+  }, [dispatch]);
```

Comment:
what's this doing here?

Reply:
https://stackoverflow.com/questions/55840294/how-to-fix-missing-dependency-warning-when-using-useeffect-react-hook

Reply:
needed it to pass Netlify

Reply:
aha.. so that's what that warning is about! 😄